Generating manpages fails with 'source is either binary or contains invalid Unicode data'

Trying to build `util-linux` from git on Debian unstable fails with an error while generating manpages:

```
  CCLD     findmnt
  CCLD     lsblk
  CCLD     mountpoint
  CCLD     fstrim
  CCLD     swapon
  CCLD     swapoff
  CCLD     fsck
  CCLD     cfdisk
  CCLD     lsns
  CCLD     lslocks
  CCLD     eject
   GEN      ar :  terminal-colors.d.5
/usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:596:in `rescue in prepare_lines': asciidoctor: FAILED: /home/glaubitz/util-linux/po-man/translations/ar/terminal-colors.d.5.adoc: Failed to load AsciiDoc document - source is either binary or contains invalid Unicode data (ArgumentError)

      raise ::ArgumentError, 'source is either binary or contains invalid Unicode data'
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:582:in `prepare_lines'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:808:in `prepare_lines'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:732:in `push_include'
        from /home/glaubitz/util-linux/tools/asciidoctor-includetracker.rb:21:in `process'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:1055:in `[]'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:1055:in `preprocess_include_directive'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:865:in `process_line'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:136:in `peek_line'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:657:in `peek_line'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:640:in `has_more_lines?'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:199:in `read_lines'
        from /home/glaubitz/util-linux/tools/asciidoctor-unicodeconverter.rb:15:in `process'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/document.rb:533:in `[]'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/document.rb:533:in `block in parse'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/document.rb:532:in `each'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/document.rb:532:in `parse'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/load.rb:84:in `load'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/convert.rb:78:in `convert'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/convert.rb:190:in `block in convert_file'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/convert.rb:190:in `open'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/convert.rb:190:in `convert_file'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/cli/invoker.rb:129:in `block in invoke!'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/cli/invoker.rb:112:in `each'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/cli/invoker.rb:112:in `invoke!'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/bin/asciidoctor:15:in `<top (required)>'
        from /usr/bin/asciidoctor:25:in `load'
        from /usr/bin/asciidoctor:25:in `<main>'
/usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/helpers.rb:126:in `encode': "\xD8" on US-ASCII (Encoding::InvalidByteSequenceError)
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/helpers.rb:126:in `prepare_source_string'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:584:in `prepare_lines'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:808:in `prepare_lines'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:732:in `push_include'
        from /home/glaubitz/util-linux/tools/asciidoctor-includetracker.rb:21:in `process'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:1055:in `[]'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:1055:in `preprocess_include_directive'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:865:in `process_line'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:136:in `peek_line'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:657:in `peek_line'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:640:in `has_more_lines?'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/reader.rb:199:in `read_lines'
        from /home/glaubitz/util-linux/tools/asciidoctor-unicodeconverter.rb:15:in `process'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/document.rb:533:in `[]'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/document.rb:533:in `block in parse'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/document.rb:532:in `each'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/document.rb:532:in `parse'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/load.rb:84:in `load'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/convert.rb:78:in `convert'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/convert.rb:190:in `block in convert_file'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/convert.rb:190:in `open'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/convert.rb:190:in `convert_file'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/cli/invoker.rb:129:in `block in invoke!'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/cli/invoker.rb:112:in `each'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/lib/asciidoctor/cli/invoker.rb:112:in `invoke!'
        from /usr/share/rubygems-integration/all/gems/asciidoctor-2.0.26/bin/asciidoctor:15:in `<top (required)>'
        from /usr/bin/asciidoctor:25:in `load'
        from /usr/bin/asciidoctor:25:in `<main>'
make[2]: *** [Makefile:19899: po4a-gen.stamp] Error 1
make[2]: Leaving directory '/home/glaubitz/util-linux'
make[1]: *** [Makefile:17872: all-recursive] Error 1
make[1]: Leaving directory '/home/glaubitz/util-linux'
make: *** [Makefile:7823: all] Error 2
(sid_amd64-dchroot)glaubitz@barriere:~/util-linux$ 
```

--- a/tools/asciidoctor-includetracker.rb
+++ b/tools/asciidoctor-includetracker.rb
@@ -17,7 +17,7 @@ module IncludeTracker
       docdir = doc.attributes["docdir"]
       path = target
       file = File.expand_path path, docdir
-      data = File.read file
+      data = File.read file, encoding: 'UTF-8'
       reader.push_include data, file, path, 1, attributes
       doc.attributes["include_dependencies"] << file
       reader

